code/form/LoginForm.php

```diff
@@ -7,6 +7,16 @@
 
 class LoginForm extends \MemberLoginForm
 {
+    private static $allowed_actions = array('cancel');
+
+    /**
+     * Action to unset the TOTP.ID session var to allow going back to the normal (email/pw) login form
+     */
+    public function cancel() {
```

Comment:
Yes, it's included again here because of my Git incompetency. I'll try & remove it.